source/patterns/@aws-solutions-constructs/aws-cloudfront-mediastore/test/integ.overrideProperties.ts

```diff
@@ -30,7 +31,10 @@ const mediaStoreContainerProps: mediastore.CfnContainerProps = {
       Effect: 'Allow',
       Principal: '*',
       Action: 'mediastore:*',
-      Resource: '*'
+      Resource: `arn:aws:mediastore:${Aws.REGION}:${Aws.ACCOUNT_ID}:container/MyMediaStoreContainer/*`,
```

Comment:
While I agree that this is probably an improvement, we can't just include other functionality in a PR that we happen to be in anyway. Especially a PR with so many changes that this unrelated change will be hard to find. Let's remove this and do it in a separate PR.

Reply:
Writing up a separate PR which will need to merged before this PR!

Reply:
https://github.com/awslabs/aws-solutions-constructs/pull/203

Reply:
Changed migrated to separate PR and merged into `main`